inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/http/InternalHttpSender.java

```diff
@@ -165,17 +156,6 @@ public List<HostInfo> getRandomHostInfo() {
 
     /**
      * send request by http
-     *
-     * @param bodies
```

Comment:
If tags of a Javadoc do not specify description info, their existence is meaningless.